src/main/java/com/solvd/webtest/pages/ios/SearchResultPageIos.java

```diff
@@ -0,0 +1,67 @@
+package com.solvd.webtest.pages.ios;
+
+import com.solvd.webtest.components.ResultMobile;
+import com.solvd.webtest.pages.base.SearchResultPageBase;
+import com.zebrunner.carina.utils.factory.DeviceType;
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.*;
+
+@DeviceType(pageType = DeviceType.Type.IOS_PHONE, parentClass = SearchResultPageBase.class)
+public class SearchResultPageIos extends SearchResultPageBase {
+    @FindBy(css = ".srp-results > li:not(.srp-river-answer)  .s-item__wrapper")
+    private List<ResultMobile> results;
+
+    @FindBy(css = ".srp-controls__control--link.srp-controls__control--link-right.srp-controls__control--link-enabled.btn")
+    private ExtendedWebElement sortingMenuBtn;
+
+    @FindBy(css = "ul.srp-sort__menu span.s-sort-label")
+    private List<ExtendedWebElement> sortOptions;
+
+    public SearchResultPageIos(WebDriver driver) {
+        super(driver);
+    }
+
+    @Override
+    public boolean isOpen() {
+        super.isOpen();
+        return getDriver().getCurrentUrl().startsWith(url);
```

Comment:
checking if page is opened by url is not a good solution, since you can have a valid url but empty page, you should check by some elements instead, or by both url and element

Reply:
I used a small hack here, I check if body is loaded in BasePage `isOpen()`

Reply:
Its left from pure Selenium version of the tests